examples/concurrency/ChannelExamples.flix

```diff
@@ -0,0 +1,182 @@
+//def allTests(): Int = {
```

Comment:
Remove this file, ChannelExamples.flix. You can move the code to some test cases, if you so desire.